app/goal_routes.py

```diff
@@ -0,0 +1,77 @@
+from flask import Blueprint, jsonify, make_response, abort, request
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from .task_routes import validate_model
+
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+######## POST GOAL ########################
+@goals_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+    try:
+        new_goal = Goal.from_dict(request_body)
+    except KeyError:
+        return jsonify({'details': 'Invalid data'}), 400
+
+    db.session.add(new_goal)
```

Comment:
Great job using `db.session.<method-name>`. Session gives us access to the follow:

- `db` is our app’s instance of SQLAlchemy.
- `session` represents our active database connection.
- By referencing `db.session` we can use SQLAlchemy’s methods to perform tasks like:
    - committing a change to a model
    - storing a new record of a model
    - deleting a record.
- By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the task model